configs/templates/ufs-weather-model-static/spack.yaml

```diff
@@ -20,16 +20,16 @@ spack:
   - zlib@1.2.11~shared
   - bacio@2.4.1
   - crtm@2.4.0~fix
-  - esmf@8.4.1+debug~shared
-  - esmf@8.4.1~debug~shared
+  - esmf@8.3.0b09+debug~shared
+  - esmf@8.3.0b09~debug~shared
```

Comment:
@Hang-Lei-NOAA can you comment on this? If we're using the updated versions in the installation on Acorn then I'd lean toward leaving those here.

Reply:
I am totally happy to revert the ufs-weather-model-static changes. As long as the default for unified-dev and skylab-dev is esmf@8.3.0b09 + mapl@2.22.0, then we have no problem.

Reply:
If we don't hear back from him in a bit let's keep the newer versions because I'm pretty sure this is what they're using on acorn for ops testing.